infertrade/api.py

```diff
@@ -204,27 +204,27 @@ def get_available_representations(name_of_algorithm: str) -> List[str]:
     def return_representations(
```

Comment:
This function needed a bit of fixing up, as it made calls with incorrect parameters to other functions in this file.